README.md

```diff
@@ -167,48 +178,110 @@ get_one_benchmark(
     level: Union[str, int],
     circuit_size: int = None,
     benchmark_instance_name: str = None,
-    opt_level: int = None,
-    gate_set_name: str = None,
-    smallest_fitting_arch: bool = None,
-)
+    compiler: str = "qiskit",
+    compiler_settings: Union[str, int] = 0,
+    gate_set_name: str = "ibm",
+    device_name: str = "ibm_washington",
+):
 ```
 The available parameters are:
   - `benchmark_name`: `"ae"`, `"dj"`, `"grover-noancilla"`, `"grover-v-chain"`, `"ghz"`, `"graphstate"`, `"portfolioqaoa"`,
                         `"portfoliovqe"`, `"qaoa"`, `"qft"`, `"qftentangled"`, `"qgan"`, `"qpeexact"`, `"qpeinexact"`,
                         `"qwalk-noancilla"`, `"qwalk-v-chain"`, `"realamprandom"`, `"su2random"`, `"twolocalrandom"`, `"vqe"`,
-                        `"wstate"`, `"shor"`, `"hhl"`, `"pricingcall"`, `"pricingput"`, `"groundstate"`, `"excitedstate"`, `"routing"`,
+                        `"wstate"`, `"shor"`, `"hhl"`, `"pricingcall"`, `"pricingput"`, `"groundstate"`, `"routing"`,
                         `"tsp"`
   - `level`: `0` or `"alg"`, `1` or `"indep"`, `2` or `"nativegates"`, `3` or `"mapped"`
-  - `circuit_size`: most of the cases this is equal to number of qubits (for some benchmarks the number of qubits is higher)
-  - `benchmark_instance_name`: `"xsmall"`, `"small"`, `"medium"`, `"large"`, `"xlarge"` (not all instances are available for each benchmark)
-  - `opt_level`: `0`, `1`, `2`, `3`
-  - `gate_set_name`: `"ibm"`, `"rigetti"`
-  - `smallest_fitting_arch`: `False`, `True`
-
-For example, in order to obtain the *5*-qubit Deutsch-Josza algorithm, use the following:
+  - `circuit_size`: for most of the cases this is equal to number of qubits 
+(all scalable benchmarks but `"qwalk-v-chain"` and `"grover-v-chain"`) while for all other the qubit number is higher
+  - `compiler`: -- `"qiskit"` or `"tket"`
+  - `compiler_settings`: Optimization level for if compiler is qiskit (`0`-`3`), Line Placement or Graph Placement if compiler is tket (`True` or `False`)

```

Comment:
this option is really confusing. `compiler_settings` should IMHO be a `Dict[str, Any]`. If you think about the future where there are more configuration options it will no longer be clear what a single integer or string stands for.
After adoption this change, the documentation here should be updated accordingly to describe the available keys ("settings") and their accepted values

Reply:
done